### PR TITLE
Fix handling logs on interactive start

### PR DESCRIPTION
**Pull request by dokshina**
When instances are started in interactive mode, colorized log writer
is used to prettify log messages. After writing some big amount of data 
to log, this function become waiting forever.
This patch fixes `ColorizedWriter.Write` function to handle number
of bytes that was wrote correctly.

### CODE REVIEW

File: cli/running/writer_test.go [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+func TestWrite(t *testing.T) {
+	t.Parallel()
+
+	assert := assert.New(t)
+
+	out := bytes.NewBuffer(nil)
+	var logs []string
+	var logBytes []byte
+	var n int
+	var err error
```
Comment (mRrvz) [inline cli/running/writer_test.go:34]: This is looks like a Pascal-style: variables are declared at the beginning, not in the place where they will be used

File: cli/running/writer.go [hunk 2]
```diff
@@ -38,23 +38,26 @@ func init() {
...
 			break
 		}
-		if err != nil {
-			return n, err
-		}
 
 		// prefix
-		if _, err := os.Stdout.Write([]byte(w.prefix)); err != nil {
+		if nPrefix, err := w.out.Write([]byte(w.prefix)); err != nil {
+			n += nPrefix
```
Comment (mRrvz) [inline cli/running/writer.go:60]: What is ```n```?

File: cli/running/writer_test.go (hunk 1 above)
Comment (dokshina) [reply cli/running/writer_test.go:34]: See below, this variables are used on each test.

File: cli/running/writer.go (hunk 2 above)
Comment (dokshina) [reply cli/running/writer.go:60]: `Write` function of `io.Writer` interface returns `n, err`, wrere `n` is number of bytes that was written.

**Review by olegrok — APPROVED** [review]

**Pull request merged**